### ISSUE TITLE
Added Rocket Sent Count to evoGUI.

### ISSUE THREAD

**Issue by KaneHart** [narc0tiq/evoGUI#35]
I would love it if such a kind and wonderful man would be willing to grab the rocket sent count and add it to evoGUI. 

If possible remove the scoreboard or give instructions how to modify the control.lua.

Thanks!

**Comment by StephanSteinert** [comment]
If anyone tell me how the rocket count is evaluated i can try to make a remote sensor in the meanwhile.
My estimation is, that the mod must be enabled before the first rocket is sent and only reacts to the on_rocket_launched event.

**Comment by silasary** [comment]
Yeah. It's stored in the Sandbox control.lua, which I believe is an
inaccessible scope. (Unless they made a remote for it?)

And even if that weren't true, with 0.13 about to hit, I'm fully expecting
the endgame to change, and Rockets to change with it.

**Comment by narc0tiq** [comment]
You could just subscribe to the [on_rocket_launched event](http://lua-api.factorio.com/0.12.35/events.html#on_rocket_launched), making a note that on the next tick you should destroy the sandbox/freeplay GUI and increment your own count by 1.

You could do it in the same tick, but I'm not sure whether the mod event will reliably fire after the scenario's event.

**Comment by jocker-il** [comment]
I know someone made a mod for that [here] (https://mods.factorio.com/mods/binbinhfr/ScoreExtended)
